### DISCUSSION TITLE
Disable Code Folding

### DISCUSSION THREAD

**Discussion by eric-hemasystems** [orbitalquark/textadept#262] [Q&A] [answered]
I've used several text editors that have code folding before and never really found it of value for my usage. I usually disable it just to remove the clutter as it tends to add a lot with all the collapse widgets. I looked through the API docs and see various things on how to customize the folding but don't see anything for just disabling it. Am I just blind? :)

**Comment by orbitalquark** [comment, marked as answer]
In your *~/.textadept/init.lua* set `lexer.folding = false`. Textadept's own *init.lua* sets it to `true` initially.

**Reply by eric-hemasystems** [reply]
Your amazing! Thanks!

**Reply by eric-hemasystems** [reply]
Small nitpick. It looks like the extra right padding to provide a place for the folding widgets is still present. Anyway to zap that padding?

**Reply by rgieseke** [reply]
I think `view.margin_width_n[3] = 0`.
